doc/lightning_tensor/device.rst

```diff
@@ -0,0 +1,117 @@
+Lightning Tensor device
+=======================
+
+The ``lightning.tensor`` device is a tensor network based device. It extends Pennylane-Lightning to run tensor network based simulations. 
+The device is built on top of the `cutensornet` from the NVIDIA cuQuantum SDK, enabling GPU-accelerated simulation of quantum tensor network evolution.
+
+A ``lightning.tensor`` device can be loaded using:
+
+.. code-block:: python
+
+    import pennylane as qml
+    dev = qml.device("lightning.tensor", wires=100)
+
+If the NVIDIA cuQuantum libraries are available, the above device will allow all operations to be performed on a CUDA capable GPU of generation SM 7.0 (Volta) and greater. 
+
+The ``lightning.tensor`` device also directly supports quantum circuit gradients using the ``parameter-shift`` method. This can be enabled at the PennyLane QNode level with:
```

Comment:
Since parameter shift is the only viable option right now anyway, is this also the default or would a user have to manually set it?

Reply:
Users have to manually set `parameter-shift`. It's not a default option.

Reply:
is it possible to make it the default or "best" for this device?

Reply:
Hey @Qottmann , it's technically fine to do that. But I would prefer to set a fast gradient method support method as `default` or `best`, probably like the `cutensornet`'s built-in gradient support (not there yet in ``cutensornet``) as `default`. ``parameter-shift`` is really slow.